pkg/kubectl/cmd/apiversions.go

```diff
@@ -51,19 +51,13 @@ func RunApiVersions(f *cmdutil.Factory, w io.Writer) error {
 		return err
 	}
 
-	apiVersions, err := client.ServerAPIVersions()
+	groupList, err := client.Discovery().ServerGroups()
 	if err != nil {
 		fmt.Printf("Couldn't get available api versions from server: %v\n", err)
```

Comment:
In general, business logic should return errors and neither output nor exit.

Additionally, normal output of commands should go to `out`, while error output should go to stderr.